src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -226,47 +226,39 @@ and a guide for using macros can be found at
 -->
 {{ MACROS___make_columns_table("meg.MEGChannels") }}
 
-Example:
-
-```Text
-name type units description sampling_frequency low_cutoff high_cutoff notch software_filters status
-UDIO001 TRIG V analogue trigger 1200 0.1 300 0 n/a good
-MLC11 MEGGRADAXIAL T sensor 1st-order grad 1200 0 n/a 50 SSS bad
-```
```

Comment:
I removed this example, because a channels.tsv example is already present below the channel types table